lms/templates/courseware/course_updates.html

```diff
@@ -35,7 +33,7 @@ <h2 class="date" id="msg-date-${index}">${update.get("date")}</h2>
 <div class="old-updates hidden toggle-visibility-element" id="old-updates">
   % for index, update in enumerate(hidden_updates):
     <article class="updates-article">
-      <h2 class="date" id="msg-date-${index + len(visible_updates)}">${update.get("date")}</h2>
+      <div class="date" id="msg-date-${index + len(visible_updates)}">${update.get("date")}</div>
```

Comment:
Same here @staubina I think its important that each Update begin with a heading element, especially since they are the first child of an ```<article>``` If we maintain a logical order here, the date would be an ```<h5>```